app/models/note.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Note < ApplicationRecord
+  belongs_to :planning_application
+  belongs_to :user
+
+  validates :entry, presence: true, length: { maximum: 500 }
```

Comment:
was this length agreed somewhere? Had a look at the A/C's but there's no suggestion of it - we don't have any length enforcements across the app so unless the partners specified this as a requirement we shouldn't enforce one without them giving it the 👍

Reply:
More of a common sense check. Probably shouldn't let our db just take a silly amount of characters but I'll confirm this exact amount